### PR TITLE
Instana addon

### PR DESCRIPTION
**Pull request by arpitn2020**
*Issue #, if available:*

*Description of changes:*
This PR updates the IBM® Instana® Addon documentation in the AddOns section. The Addon consists of separate public [NPM packages](https://www.npmjs.com/package/@instana/aws-eks-blueprint-addon), and their source code can be found [here](https://github.com/instana/instana-eks-blueprint-addon/).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by arpitn2020** [comment]
Here is the documentation PR
https://github.com/aws-quickstart/cdk-eks-blueprints/pull/708

File: lib/instana-construct/index.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import * as blueprints from "@aws-quickstart/eks-blueprints";
+import { loadYaml } from "@aws-quickstart/eks-blueprints/dist/utils";
+import * as cdk from "aws-cdk-lib";
+import { InstanaOperatorAddon } from "@instana/aws-eks-blueprint-addon";
+
+export const instanaProps = {
+  zone: {
+    name: "<INSTANA_ZONE_NAME>", // Mandatory Parameter
```
Comment (elamaran11) [inline lib/instana-construct/index.ts:8]: Could we avoid temporary hardcoding like this and have an environment variable populated and thats use here? 
Also we should have a documentation talking about how to extract Agent Key, Endpoint host url/port, zone etc.

File: package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "typescript": "^5.0.4"
   },
   "dependencies": {
-    "@aws-quickstart/eks-blueprints": "^1.8.1",
+    "@aws-quickstart/eks-blueprints": "latest",
```
Comment (elamaran11) [inline package.json:27]: Please remove this, we generally point to specific version in patterns.

File: package.json [hunk 3]
```diff
@@ -36,10 +36,11 @@
     "@snyk-partners/snyk-monitor-eks-blueprints-addon": "^1.1.0",
     "aws-cdk": "2.81.0",
     "eks-blueprints-cdk-kubeflow-ext": "0.1.9",
-    "source-map-support": "^0.5.21"
+    "source-map-support": "^0.5.21",
+    "@instana/aws-eks-blueprint-addon": "^1.0.2"
   },
   "overrides": {
-    "@aws-quickstart/eks-blueprints": "^1.8.1",
+    "@aws-quickstart/eks-blueprints": "latest",
```
Comment (elamaran11) [inline package.json:43]: Please remove this, we generally point to specific version in patterns.

**Review by elamaran11 — COMMENTED** [review]
@arpitn2020 Thankyou for an amazing work on the PR. The PR is missing a documentation page like [this](https://github.com/aws-samples/cdk-eks-blueprints-patterns/blob/main/docs/patterns/multi-account-monitoring.md). The documentation should talk about how to get these environment paremeters like Zone, url, port etc and that should be rendered to the solution via a environment variable. You can refer other docs. This approach will help our customers to seamlessly to use Instana as a solution.

File: lib/instana-construct/index.ts (hunk 1 above)
Comment (arpitn2020) [reply lib/instana-construct/index.ts]: Hi @elamaran11 . Thanks for the feedback. I made changes to remove hardcoding and added detailed documentations to get value of Agent Key, Endpoint host url/port etc.

File: package.json (hunk 2 above)
Comment (arpitn2020) [reply package.json]: done.

File: package.json (hunk 3 above)
Comment (arpitn2020) [reply package.json]: done

**Review by elamaran11 — DISMISSED** [review]
@arpitn2020 LGTM. Thanks for the PR.
@shapirov103 FYI

**Comment by arpitn2020** [comment]
@elamaran11 The pipeline (cdk synth) is failing because it requires specific environment variables to be set before it can be executed. This has been documented in the pattern documentation.

**Comment by arpitn2020** [comment]
@elamaran11 Please review the changes.

**Review by elamaran11 — APPROVED** [review]
LGTM

**Comment by elamaran11** [comment]
@arpitn2020 We would need Instana access to complete the validation before we can move towards review and merge of addons doc PR. Could you or Mike provide me access to Instana portal.

**Pull request merged**